code/modules/mob/living/carbon/xenomorph/life.dm

```diff
@@ -103,6 +103,9 @@ adjustFireLoss(-(maxHealth / 70 + 0.5 + (maxHealth / 70) * recovery_aura/2)*(m))
 				if(M.loc != src)
 					continue
 				M.forceMove(loc)
+				if(isliving(M))
+					var/mob/living/L = M
+					L.blinded = FALSE
```

Comment:
Since this code is repeating in each instance, it could become a proc under `/mob/` to simplify things.
Also this can act as an unintended cure for blindness. Make the blind removal check for other conditions.

Reply:
Sure, I'll run a check for the blind disability.

The only thing that this might interfere with is blind stacks, but their blind gets quickly reapplied anyways if the victim still has stacks.